Safety/AttitudeManager/Src/attitudeStateClasses.cpp

```diff
@@ -101,10 +115,24 @@ attitudeState& sensorFusionMode::getInstance()
 
 void PIDloopMode::execute(attitudeManager* attitudeMgr)
 {
-
-    CommandsForAM *PMInstructions = fetchInstructionsMode::GetPMInstructions();
+    CommandsForAM *PMInstructions = nullptr;
     SFOutput_t *SFOutput = sensorFusionMode::GetSFOutput();
 
+    PID_Output_t *pidOut = nullptr;
+    if(fetchInstructionsMode::isAutonomous())
+    {
+        PMInstructions = fetchInstructionsMode::GetPMInstructions();
+        //TODO: Run controls module with PMInstructions
+    }
+    else
+    {
+        PPM_Instructions_t *teleopInstructions = fetchInstructionsMode::GetTeleopInstructions();
+        _PidOutput = getPIDFromControls(teleopInstructions, SFOutput);
```

Comment:
I'll need to update this function name. Is runControlsAndGetPWM(...) what we're calling it?

Reply:
getPIDFromControls seems a lot nicer than runControlsAndGetPWM, .. especially since it's a PID and not PWM output